### ISSUE TITLE
Large-Margin Softmax Loss for Convolutional Neural Networks

### ISSUE THREAD

**Issue by joyhuang9473** [number9473/nn-algorithm#38]
# Large-Margin Softmax Loss for Convolutional Neural Networks #

- Author: Weiyang Liu, Yandong Wen, Zhiding Yu, Meng Yang
- Origin: [https://arxiv.org/abs/1612.02295](https://arxiv.org/abs/1612.02295)
- Related:
  - [https://github.com/wy1iu/LargeMargin_Softmax_Loss](https://github.com/wy1iu/LargeMargin_Softmax_Loss)
  - https://github.com/luoyetx/mx-lsoftmax
  - [基于Caffe的Large Margin Softmax Loss的实现（上）](http://www.miaoerduo.com/deep-learning/%E5%9F%BA%E4%BA%8Ecaffe%E7%9A%84large-margin-softmax-loss%E7%9A%84%E5%AE%9E%E7%8E%B0%EF%BC%88%E4%B8%8A%EF%BC%89.html)
  - [基于Caffe的Large Margin Softmax Loss的实现（中）](http://www.miaoerduo.com/deep-learning/%E5%9F%BA%E4%BA%8Ecaffe%E7%9A%84large-margin-softmax-loss%E7%9A%84%E5%AE%9E%E7%8E%B0%EF%BC%88%E4%B8%AD%EF%BC%89.html)
